src/pages/Profile/subpages/Friends/FriendCard.js

```diff
@@ -63,11 +53,25 @@ function FriendCard({ friend, currentFriend }) {
     }
     handleAge(years);
   }
+  function splitLongUsername(username) {
```

Comment:
@GreenWithMV I'm thinking that CSS truncate may be better here if this is mainly around the size of the username in the UI; but another approach I've seen on some UI's is to use the user's initials when things are really minimized.